spec/models/manageiq/providers/redhat/infra_manager/ems_cluster_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe ManageIQ::Providers::Redhat::InfraManager::EmsCluster do
+  context "#upgrade" do
+    before do
+      @ems = FactoryGirl.create(:ems_redhat_with_authentication)
+      @cluster = FactoryGirl.create(:ems_cluster_redhat, :ems_id => @ems.id)
+      my_server = double("my_server", :guid => "guid1")
+      allow(MiqServer).to receive(:my_server).and_return(my_server)
```

Comment:
What ```receive(:my_server)``` stands for ? it doesn't seems like a method name

Reply:
It is a method name, it is used here to put a task into MiqQueue: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_runner_workflow.rb#L72